frontend/sw360-portlet/src/main/webapp/html/projects/view.jsp

```diff
@@ -321,17 +305,64 @@
                         }
                     ],
+                    "pageLength": 10,
                     "search": {smart: false},
                     "columns": [
                         {title: "Project Name", data: "name", render: {display: renderProjectNameLink}},
-                        {title: "Description", data: "description"},
-                        {title: "Project Responsible", data: "responsible"},
-                        {title: "State", data: "state"},
+                        {title: "Description", data: "desc", render: {display: renderDescription}},
+                        {title: "Project Responsible", data: "resp", render: {display: renderProjectResponsible}},
+                        {title: "State", data: "state", render: {display: renderStateBoxes}},
                         {title: "<span title=\"Release clearing state\">Clearing Status</span>", data: "clearing"},
                         {title: "Actions", data: "id", render: {display: renderProjectActions}}
                     ],
-                    "autoWidth": false
+                    "order": [[0, 'asc']],
+                    "lengthMenu": [[10, 25, 50, 100, -1], [10, 25, 50, 100, "All"]],
+                    "autoWidth": false,
+                    "fnDrawCallback": function (oSettings) {
+                        loadClearingStateSummaries();
+                    }
                 });
+
+                return projectsTable;
+            }
+
+            function renderStateBoxes(state, type, row) {
+                var projectStateBackgroundColour = '<%=PortalConstants.PROJECT_STATE_INACTIVE__CSS%>';
+                var clearingStateBackgroundColour = '<%=PortalConstants.CLEARING_STATE_UNKNOWN__CSS%>';
+
+                if(state && state === 'ACTIVE') { // -> green
+                    projectStateBackgroundColour = '<%=PortalConstants.PROJECT_STATE_ACTIVE__CSS%>';
+                }
+
+                if (row.cState) {
+                    switch (row.cState) {
+                        case 'CLOSED': // -> green
+                            clearingStateBackgroundColour = '<%=PortalConstants.CLEARING_STATE_CLOSED__CSS%>';
+                            break;
+                        case 'IN_PROGRESS': // -> yellow
+                            clearingStateBackgroundColour = '<%=PortalConstants.CLEARING_STATE_INPROGRESS__CSS%>';
+                            break;
+                        case 'OPEN': // -> red
+                            clearingStateBackgroundColour = '<%=PortalConstants.CLEARING_STATE_OPEN__CSS%>';
+                            break;
+                    }
+                }
+
+                var box_PS = "<div class=\"stateBox " + projectStateBackgroundColour + " capsuleLeft \" title=\"Project state: " + "Test Title" + "\"> PS </div>";
+                var box_CS = "<div class=\"stateBox capsuleRight " + clearingStateBackgroundColour + "\" title=\"Project clearing state: " + "Test Title" + "\"> CS </div>";
```

Comment:
Inconsistend ordering of classes makes it hard to read

Reply:
sorted the class names